kivy/uix/textinput.py

```diff
@@ -835,7 +856,186 @@ def _set_undo_redo_bkspc(self, ol_index, new_index, substring, from_undo):
         #reset redo when undo is appended to
         self._redo = []
 
-    def do_cursor_movement(self, action):
+    def _move_word_left(self, index=None):
```

Comment:
Could you please rename it to _move_cursor_...? would be more clear while reading the code.